src/main/java/org/quiltmc/loader/impl/QuiltLoaderImpl.java

```diff
@@ -159,7 +171,7 @@ public Path getConfigDir() {
 			try {
 				Files.createDirectories(configDir);
 			} catch (IOException e) {
-				throw new RuntimeException("Creating config directory", e);
+				throw new RuntimeException("Failed to create config directory", e);
```

Comment:
Hopefully not too big of an interjection, could you include the stringified path in the stacktrace so we know where the mods folder creation has failed if the property is passed